vwr/driver.py

```diff
@@ -148,6 +177,8 @@ def _receive(self):
             response = message.decode('utf-8').strip()
         except socket.timeout:
             response = None
-            self._connect()
+            if self.connected:
+                self.connected = False
+                Timer(self.initialDelay, self._reconnect).start()
```

Comment:
I would just call `self._reconnect()` here. Keep all that Timer mess inside that function.